src/bindings/c/include/openvino/c/ov_layout.h

```diff
@@ -0,0 +1,47 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/**
+ * @brief This is a header file for ov_layout C API
+ *
+ * @file ov_layout.h
+ */
+
+#pragma once
+
+#include "openvino/c/ov_common.h"
+
+typedef struct ov_layout ov_layout_t;
+
+// Layout
+/**
+ * @defgroup layout layout
+ * @ingroup openvino_c
+ * Set of functions representing of Layout.
+ * @{
+ */
+
+/**
+ * @brief Create a layout object.
+ * @ingroup layout
+ * @param ov_status_e a status code, return OK if successful
+ */
+OPENVINO_C_API(ov_status_e) ov_layout_create(ov_layout_t** layout, const char* layout_desc);
```

Comment:
maybe ov_layout_from_string? or even ov_creaete_layout_from_string?

Reply:
Will it be more concise if we only have a single such api?